### PR TITLE
Capstone Project 1: Add mobile version and Desktop Version 

### PR DESCRIPTION
**Pull request by pasytchangwa**
## Added Mobile Version and Desktop Version with a dynamic section

- Added mobile version
- Added Desktop version
- Added Mobile Menu
- Added a dynamic featured speakers section
- Added link interaction
- Added More item bar on the homepage

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                alt="project logo" class="logo"></a>
+          </div>
+          <ul class="nav-items">
+            <li class="nav-item"><a href="./about.html" class="nav-link change">about</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">program</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">join</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">sponsor</a></li>
+            <li class="nav-item"><a href="#" class="nav-link change">news</a></li>
+          </ul>
+          <span class="campaign"><a href="#" class="campaign-link">CSA campaign</a></span>
```
Comment (Dipeshtwis) [inline index.html:38]: - [x] From 320px to 450px, there is some white space in the right side which affect all the section in `index.html` file. Please fix that

### Your design

Screenshot from 2021-07-08 19-48-24

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @pasytchangwa, 

:+1: This is good to make it this far, your project is really nice I personally loved your project, congratulations on the good work you have done so far.
:+1: Used semantic HTML. 
:+1: Linters are working properly. :white_check_mark: 
:+1: UI Design is awesome. :green_heart: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

- [x] Please make a video presenting your capstone and put the link in your README file or PR description so that code reviewer will see and review. 
 - The video contains a clear and objective description about the project in professional English. 
 - The video contains a demo of the project, showing the pages in different screen sizes and also highlight about a piece of the code the you finds interesting or is proud of making.
 
 **_Also check the comments under the review_**
 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (pasytchangwa) [reply index.html:38]: Done

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Sylvestre, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewers. There is nothing else to say other than... it's time to merge it  👍.
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
__As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).__

**Pull request merged**